packages/chusho/src/components/CDialog/CDialog.ts

```diff
@@ -62,6 +62,8 @@ export default Vue.extend({
 
   methods: {
     activate(): void {
+      if (this.$chusho.openDialogs.includes(this)) return;
```

Comment:
I think we should rather rely on a local boolean variable defining if the component has been activated or not and skip activating it again based on it. Also do you think you could add a non-regression test around this?